ansible/roles/master/templates/scheduler.j2

```diff
@@ -4,4 +4,4 @@
 # default config should be adequate
 
 # Add your own!
-KUBE_SCHEDULER_ARGS="--kubeconfig={{ kube_config_dir }}/scheduler.kubeconfig"
+KUBE_SCHEDULER_ARGS="--kubeconfig={{ kube_config_dir }}/scheduler.kubeconfig {% if groups['masters']|length > 1 %}--leader-elect=true{% endif %}"
```

Comment:
Very good point, I'll look it up and test it out. I naively assumed that it being 'off' by default had some purpose.